schema/dataKeys.json

```diff
@@ -380,6 +388,10 @@
           "description": "The environment for the CDN configuration e.g. qa"
         }
       }
+    },
+    "registrySecret": {
```

Comment:
I personally think this fits better under mapping. It only has to do with the component repos, nothing else

Reply:
I can easily move it. But you could say the same thing about a number of other things: cnd, signing, pyxis, even github. All of these related to the components in mapping. Do you still think I should move it under mapping?

Reply:
I see what you're saying, but pyxis, github, cdn are all their own systems. This is a key used only for the quay repos defined in the mapping. If we were to follow the style of the other keys, this would be quay.registrySecret (which I don't like either). This is why I am saying it should go in the mapping. Also, if we some day support more than one, it will be a registrySecret per component, which would have to be in the mapping

Reply:
OK, I guess it makes sense. Moved.